data/mode.txt

```diff
@@ -1 +1 @@
-secret
+extended
```

Comment:
Yeah, it is. It should be extended, no idea when it was changed to secret. It's all good.